dynamical_scaling.c

```diff
@@ -0,0 +1,101 @@
+#include<stdio.h>
+#include<stdlib.h>
+typedef struct vector

```

Comment:
Take a new struct VectorHead , which will store vector* vec ; and printType function pointer. 
You will pass a object of VectorHead to all insert,display etc methods